scripts/fetch_related_shas.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+import os
+
+from github import Github
+
+from common import extract_related_prs, get_syncwith_issue
+from utils import assign_output
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Process SyncWith PRs")
+    parser.add_argument("repo_name", help='Repository in format "Org/Repo"')
+    parser.add_argument("pr_number", type=int, help="Pull request number")
+    args = parser.parse_args()
+
+    token = os.environ.get("CI_TOKEN")
```

Comment:
@knazarov Can you help me here a bit. In near future we will need to create self-hosted runners for zk, because we are running out of memory for some test on the github runners. Once we do it, will this be safe?

Reply:
why do we need CI_TOKEN here?

Reply:
> why do we need CI_TOKEN here?
You're right, no need. Removed it